### ISSUE TITLE
holding down mouse and moving outside of brush size slider, paints & adjusts brush size

### ISSUE THREAD

**Issue by unitycoder** [unitycoder/UnityMobilePaint#17]
it should either stop dragging slider when outside, or dont paint while still dragging slider
http://forum.unity3d.com/threads/released-mobile-paint-with-multi-touch.262645/page-4#post-2483179

**Comment by unitycoder** [comment]
temporary fix for pc:
// FIND LINE
`if (eventSystem.IsPointerOverGameObject()) return;`

// ADD THIS LINE AFTER IT
`if (eventSystem.currentSelectedGameObject!=null) return;`

**Comment by unitycoder** [comment]
temporary fix for mobiles:
// FIND LINES

```
while (i < Input.touchCount)
{
     touch = Input.GetTouch(i);
     if (eventSystem.IsPointerOverGameObject(touch.fingerId)) return;
     i++;
}
```

 // ADD THIS LINE AFTER THOSE LINES ( after last } )
`if (eventSystem.currentSelectedGameObject != null) return`

**Comment by unitycoder** [comment]
fixed in the coming v1.96 update
